src/app/creative-team/dashboard/cards/cards.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-dashboard-cards',
```

Comment:
try to have the prefix like af-dashboard-cards. Since Seal team used prefix in the selector to have unique selector with prefix.